app/User.php

```diff
@@ -173,12 +187,12 @@ public function routeNotificationForMail()
 
     public function getAuthIdentifierName()
     {
-        return 'uid';
+        return 'id';
     }
 
     public function getAuthIdentifier()
     {
-        return $this->uid;
+        return $this->id;
```

Comment:
What does this do?

Reply:
Fixes viewing actions on some resources.